src/components/editor/editor-component.css

```diff
@@ -17,15 +17,54 @@
   }
```

Comment:
All the CSS rules need to be scoped to the top level component element, in this case it is the '#editor-container'. See https://github.com/BigBlockDataChain/morbo/blob/master/src/components/toolbar/search-component.css for the scoping of CSS rules to the '#search' element.

Reply:
b56845d scopes all CSS rules to top-level element